tests/conftest.py

```diff
@@ -151,3 +160,48 @@ def mock_is_item_available():
     with mock.patch(path) as mock_is_item_available:
         mock_is_item_available.return_value = True
         yield mock_is_item_available
+
+
+@pytest.fixture()
```

Comment:
This is not needed as it's used as a function not as a fixture :)

Reply:
done :)